src/paulscode/android/mupen64plusae/input/map/VisibleTouchMap.java

```diff
@@ -551,12 +551,12 @@ private void loadFpsIndicator( Profile profile )
      */
     private void loadAutoHoldImages( Profile profile, String name )
     {
-        Matcher matcher = Pattern.compile( "([^-]*)-hold(.*)" ).matcher( name );
-        if( matcher.groupCount() < 3 )
+        if ( !name.contains("-hold") )
             return;
         
-        String group = matcher.group( 1 );
-        String hold = matcher.group( 2 );
+        String[] fields = name.split( "-hold" );
+        String group = fields[0];
+        String hold = fields[1];
         
```

Comment:
Ugh, I really need to learn my way around GitHub PRs – keep hitting the wrong reply button. None of the images loaded correctly since Matcher always returned a group count of 2, and Pattern.compile.matcher seemed like overkill for a simple field delimiter anyway so I just switched it out to split. All of the images load correctly now.

However, one example is buttonL-holdL.png, and another is groupAB-holdB.png.